### ISSUE TITLE
Test Receiver AX150

### ISSUE THREAD

**Issue by cpbridges** [drmpeg/gr-dvbs2#2]
Hi Ron,
I'm validating correct operation of the DVBS2 code with an Opticum AX150 DVBS2 Receiver and essentially trying to emulate an Astra1 channel by going in at IF using a PlutoSDR and bypassing the LNB. I've calculated I need to be at 979MHz for Channel 3 (10.729G - 9.750G). The photo attached shows the transponder freq, bitrate, and FEC. On the lower channels, no 22 kHz tone is required.

I also used the dtv-tools / dvbs2 calculator to set the symbol rate which is doubled for the sample rate in the flowgraph:
```
16APSK, pilots on
coderate = 2/3,  BCH rate = 10, ts rate = 11328299.172959
```
I've tried a direct cable and also via some small antennas. I'm guessing the 1mW RF from the Pluto should still work on some tiny monopoles. I'm only ever getting 70-90% signal, and 0% quality on my receiver. Any pointers?

73s Chris

Refs: https://translate.google.co.uk/translate?hl=en&sl=de&u=https://www.opticum-gmbh.de/hdtv-satreceiver/opticum-hd-ax-150

![img_2885](https://user-images.githubusercontent.com/2065141/36609191-efdb909c-18c3-11e8-8b55-923ad06e62f8.JPG)

**Comment by csete** [comment]
Have you seen what is coming out of your PlutoSDR (spectrum and video)?

As far as I can see, you are using 22 Msps, which in my experience is practically impossible with the PlutoSDR running as an IQ device. The highest rate I have accomplished in a sustainable way over USB2 is 20 Msps using 2x8 bit sample resolution. So, if Pluto uses 16 bit samples you should stay below 10 Msps.

It's hard to know what "signal quality" means in your receiver, but it sounds reasonable that frequent sample drops will result in low signal quality.

**Comment by cpbridges** [comment]
Yes, I'm operating at 22 M SYMBOLS! ps - got muddled. Need to reduce that and retest. 73s.

Edit: Actually, it was 22 M samples ps - how joyous Friday's are to our brains. Screenshots inc.

**Comment by cpbridges** [comment]
Here's my scope shots as sensible rates. I have a better one in the cleanroom that can confirm.
Top left 5 Msample/s 3s scan time, top right 0.5s scan time.
Bottom left 10 Msample/s 3s scan time, bottom right - carrier/LO still ON when flowgraph has stopped (bug?).

![plutosdr dvbs2](https://user-images.githubusercontent.com/2065141/36613682-23f2ad12-18d2-11e8-89e6-ffd8bb0b73cc.png)

Question is: will a standard receiver pick up these lower rates?

I think I'm still unclear about the dvb_tool output and if the ts rate = symbol rate.

Have a good weekend! C

**Comment by csete** [comment]
Well, at least the spectrum looks ok.

It looks like the device is not properly shut down when the flowgraph is stopped. Could be both gnuradio and/or driver.

Not sure what dvb_tool is... do you mean dvbs2rate? It takes the symbol rate as input and calculated TS bitrate for various modulations and FEC rates.

**Comment by drmpeg** [comment]
Spectrum looks horrible. You are underflowing severely. Try 3 Msyms/s (6 Msps). Most receivers will work down to around 1 Msyms/s.
